applications/visit/profile.json

```
@@ -0,0 +1,18 @@
+{
+    "name": "visit",
```

let's include the version in the name too
```suggestion
    "name": "visit-3.3.3",
```